tests/php/Extension/FluentSiteTreeExtensionTest.php

```diff
@@ -255,6 +255,24 @@ public function testAddLocalePrefixToUrlSegment($localeCode, $fixture, $expected
         $this->assertSame($expected, $segmentField->getURLPrefix());
     }
 
+    public function testHomeNotVisibleOnFrontend()
+    {
+        FluentState::singleton()->setIsFrontend(true);
+
+        $page = Page::get()->filter('URLSegment', 'home')->first();
+
+        $this->assertNull($page);
+    }
+
+    public function testHomeVisibleInCMS()
+    {
+        FluentState::singleton()->setIsFrontend(false);
```

Comment:
I think these tests would be more verbose if we were explicitly setting the config props you're adding that affect these results. Would you mind adding them?

Reply:
Thanks, @robbieaverill . Just to check that I'm on the right track here, you mean doing something like this?

```
    public function testHomeVisibleInCMS()
    {
        Config::nest();

        Config::modify()->set(DataObject::class, 'cms_publish_required', false)

        ... perform tests

        Config::unnest();
    }
```

Reply:
Yep exactly only you don't need the nesting